cvat/apps/documentation/data_on_fly.md

```diff
@@ -16,26 +16,10 @@ This method of working with data allows:
 - reduce the task creation time.
 - store data in a cache of limited size with a policy of evicting less popular items.
 
-## Prepare meta information
-
-Different meta information is collected for different types of uploaded data.
-
-### Video
-
-For video, this is a valid mapping of key frame numbers and their timestamps. This information is saved to `meta_info.txt`.
-
-Unfortunately, this method will not work for all videos with valid meta information.
+Unfortunately, this method will not work for all videos with valid manifest file.
```

Comment:
@Marishka17 , could you please describe "old way". Whey somebody reads the document, it is not obvious. Also there are some typos in the beginning of the document. For example, "follows: Minimum" and "a necessary chunks"

Reply:
I will fix it, thanks